akka-actor/src/main/scala/akka/actor/FSM.scala

```diff
@@ -713,7 +713,8 @@ trait FSM[S, D] extends Actor with Listeners with ActorLogging {
       currentState = nextState
 
       def scheduleTimeout(d: FiniteDuration): Some[Cancellable] = {
-        import context.dispatcher
+        import scala.concurrent.ExecutionContext
+        implicit val ec: ExecutionContext = context.dispatcher
```

Comment:
rather as a field perhaps?

Reply:
done with bd83098699e850856454b2a85aae71a3cce08b46 and cf66433ceaf9258b6f5d7de6f36bc54a44d1cc44